### PR TITLE
ENG-2537 Adds DAG Layout Test Cases

### PR DESCRIPTION
**Pull request by agiron123**
## Describe your changes and why you are making these changes
This PR adds Manual QA test cases to check that our nodes and edges are being rendered properly in the workflow details page.

## Related issue number (if any)
ENG-2537

## Loom demo (if any)
(These workflows exercise test cases, not fix them yet 🙂 )

Screenshots:
March Madness Layout Test:
image

Fanout and General Layout Tester:
image

## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: manual_qa_tests/workflows/dag_layout_test.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import aqueduct as aq
+
+NAME = "dag_layout_test4"
```
Comment (likawind) [inline manual_qa_tests/workflows/dag_layout_test.py:3]: let's have names starts with `succeed_` to follow the guidelines in `README.md`

File: manual_qa_tests/workflows/dag_layout_test.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+DESCRIPTION = """
+    * Workflows Page: TODO: Describe layouts here.
```
Comment (likawind) [inline manual_qa_tests/workflows/dag_layout_test.py:5]: TODO

File: manual_qa_tests/workflows/dag_layout_test.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+def op4(param):
+    return param
+
+
+@aq.op
+def op5(param):
+    return param
+
+
+@aq.op
```
Comment (likawind) [inline manual_qa_tests/workflows/dag_layout_test.py:34]: I'd suggest change a few of them from `op` to `metrics` or `checks` given there are known issue and planned changes to improve the positioning of these nodes

File: manual_qa_tests/initialize.py [hunk 4]
```diff
@@ -18,6 +20,8 @@
 # when adding new deployments, keep the order of `fail`, `warning`, and `succeed`
 # such that the UI would approximately show these workflows in reverse order.
 WORKFLOW_PKGS = [
+    dag_layout_test,
+    march_madness_dag_layout_test,
     check_status_test,
     fail_bad_check,
```
Comment (likawind) [inline manual_qa_tests/initialize.py:26]: I know I also forgot to follow the ordering here, but would appreciate if you could help sorting them in fail - warning - success order :)

**Review by likawind — APPROVED** [review]
Great work! A few nit picky comments on naming, sorting, etc.

File: manual_qa_tests/initialize.py (hunk 4 above)
Comment (agiron123) [reply manual_qa_tests/initialize.py:27]: done!

File: manual_qa_tests/workflows/dag_layout_test.py (hunk 3 above)
Comment (agiron123) [reply manual_qa_tests/workflows/dag_layout_test.py:34]: Sounds good, I've added some to the workflow.

**Pull request merged**